### PR TITLE
Add ING

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Looks like they also support 2FA through some kind of smartphone app. Is that correct?
If so you can add `software: Yes` as well :wink:

**Comment by nicolasleger** [comment]
I can not confirm the smartphone app, but I'm sure for the token generator key.

**Comment by Carlgo11** [comment]
Could you add it anyway?

**Comment by mxxcon** [comment]
Perhaps this should also be renamed to ING(Luxembourg)? Or somehow else differentiate it from ing.com proper? Unless they all share the same login system then perhaps list the main .com site?

File: _data/banking.yml [hunk 1]
```diff
@@ -255,6 +255,13 @@ websites:
       img: huntingtonnationalbank.png
       tfa: No
 
+    - name: ING
```
Comment (Carlgo11) [inline _data/banking.yml]: As @mxxcon requested earlier, please change the name to ING (Luxembourg)

**Comment by nicolasleger** [comment]
Renaming done

**Pull request merged**